kratos/utilities/read_materials_utility.h

```diff
@@ -197,6 +198,28 @@ class KRATOS_API(KRATOS_CORE) ReadMaterialsUtility
     ///@name Private Operations
     ///@{
 
+    /**
+     * @brief This method creates a property from configuration parameters
+     * @param Data The parameters containing all the configurations of the materials
+     * @param pNewProperty The pointer to the new property created
+     */
+    void CreateProperty(
+        Parameters Data,
+        Properties::Pointer& pNewProperty
```

Comment:
if we add here 
treat_as_subproperty 

the logics will become easier

Reply:
OK